events/app.py

```diff
@@ -49,7 +42,7 @@ def add_event():
         }
         # TODO(cmei4444): Athenticate user and verify that user has event
         # editing access
-        current_time = datetime.datetime.now()
+        current_time = datetime.datetime.utcnow().isoformat(" ", "seconds")
```

Comment:
same comment about parameter labels on this call. That " " is hard to read/understand what it means.

Reply:
Clarified in 5fdd6a3